src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -22,15 +22,14 @@
 
 (defmethod ct/report #?(:clj ::shrunk :cljs [::ct/default ::shrunk]) [m]
   (newline)
-  (println "Tests failed, smallest case:" (pr-str (-> m :shrunk :smallest))
-           "\nSeed" (:seed m)))
+  (println (str "Tests failed, smallest case: " (pr-str (-> m :shrunk :smallest))
+                "\nSeed: " (:seed m))))
```

Comment:
this is because the output looks nicer this way?

Reply:
At least to my tastes, yeah that's all.